bitrise/template_util.go

```diff
@@ -23,14 +24,25 @@ type TemplateDataModel struct {
 }
 
 func getEnv(key string, envList envmanModels.EnvsJSONListModel) string {
-	if len(envList) > 0 {
+	envfilePath := os.Getenv(envfile.DefaultEnvfilePathEnv)
```

Comment:
99% of the cases we don't need the env file, could we reshuffle this function to only interact with env file if needed (i.e.: if env var value is empty in envList [+ if key presents in the cleared envs list])?